measurements/api/measurements.py

```diff
@@ -200,31 +201,33 @@ def get_measurement(measurement_id, download=None):
                 'attachment', filename=filename)
     return response
 
-def input_filter(q, input_, domain, test_name):
+def input_cte(input_, domain, test_name):
```

Comment:
input_cte appears not to be covered in https://travis-ci.org/ooni/api/builds/581319654#L2112  - is it worth testing?

Reply:
I added a basic check to the smoke test. Without fixtures in the DB though it's hard to have a really useful test in here though.